games/minetest/Portfile

```diff
@@ -8,9 +8,9 @@ compiler.cxx_standard   2011
 compiler.thread_local_storage   yes
 
 github.setup            minetest minetest 5.6.1
-revision                0
+revision                2
```

Comment:
the `revision` should be `1` here. You are making changes to the Portfile (without changing the `version`) that will result in a different build; that warrants increasing the `revision` by one (i.e., from `0` to `1`) not by two.

Reply:
after making this last change (at least I don't see anything else anymore, perhaps others do...), please squash your commits so that there is only one left.

Reply:
@reneeotten - THX & OK

However, let me ask _honestly_? You fell into the little trap of my social experiment (or poor man's [NLP](https://en.wikipedia.org/wiki/Neuro-linguistic_programming)) and obviously insist to "enforce" (sorry ;) a step of just one i.e. "1" although there should be 65335 in 16 bit numbering or maybe more? 

IMO this is **strangely funny**, quite correct, utterly true and **nevertheless feasible**.

_Maintainer in the making_ will oblige happily and I will certainly change both the new Portfile and this PR to Revision "1" i.e. 1️⃣ in particular. Yes, Sir.
🤣 

### **quot capita tot sensus**

For everybody there might be a life waiting outside GitHub, I presume. Happy coding and have fun

🌻

Reply:
> after making this last change (at least I don't see anything else anymore, perhaps others do...), please squash your commits so that there is only one left.

_NOTE & NB: spell check corrects to "**any more**" instead of "anymore" on a Firefox browser with en_GB locale._

IMO the final PR and commit both **can wait until next year** and might accept last remarks until then, I presume.

Strange things seem to happen, I presume. Nevertheless a (very?) good learning expertise.

Reply:
NB

The experts might get a good laugh at me but GitHub Desktop for macOS can be (somewhat) useless.

Why did I edit and combine the text of all commits before the squash only to find GitHub sqashing all into the one older commit below and making a new concatenated text summary by its own while loosing the new commit somehow? 

RTM and you fault as a greenhorn they will say. My mileage would vary.

Livelong learning and software tools are never perfect, I presume.

Have fun and a a more reliable tool..

🧮